CHANGELOG.md

```diff
@@ -1,8 +1,16 @@
 # Changelog
 
-## 1.24.0-dev
+## 1.24.0
 * Feature - Configurable poll duration for container stats [@jcbowman](https://github.com/jcbowman) [#1646](https://github.com/aws/amazon-ecs-agent/pull/1646)
 * Feature - Add support to remove containers and images that are not part of ECS tasks [#1752](https://github.com/aws/amazon-ecs-agent/pull/1752)
+* Feature - Add ARM build script [#1747](https://github.com/aws/amazon-ecs-agent/pull/1747)
+* Feature - Add configurable prometheus endpoint for agent metrics [#1743](https://github.com/aws/amazon-ecs-agent/pull/1745)
+* Feature - Add Host EC2 instance Public IPv4 address to container metadata file [#1730](https://github.com/aws/amazon-ecs-agent/pull/1730)
+* Enhancement - Docker SDK migration replacing go-dockerclient [#1743](https://github.com/aws/amazon-ecs-agent/pull/1743)
+* Enhancement - ECR image pull retry changed to 5 times [#1721](https://github.com/aws/amazon-ecs-agent/pull/1721)
+* Enhancement - Propagating Container Instance and Task Tags to Task Metadata endpoint [#1720](https://github.com/aws/amazon-ecs-agent/pull/1720)
+* Enhancement - Fetch AMI ID dynamically [#1754](https://github.com/aws/amazon-ecs-agent/pull/1754)
+* Bug - Fixed the edge case for stopping container when the StartContainer API call is timed out [#1749](https://github.com/aws/amazon-ecs-agent/pull/1749)
```

Comment:
This PR is not merged, please remove this line

Reply:
Will remove this